### DISCUSSION TITLE
"Can you hack for me?"

### DISCUSSION THREAD

**Discussion by PatheticMustan** [Prodigy-Hacking/ProdigyMathGameHacking#170] [(A) Anything]
# We do not, and will not, hack your account for you

The goal of this repository is to host a large amount of hacks, so that Prodigy can fix the bugs, and make itself the best that it can be.

Not only do we not care about your accounts, but we really don't want to spend time hacking some random stranger's account.

We made it as easy as possible to hack, with [very simple instructions], and even a discord server to ask for help, should you get stuck on any part.
Not only is it annoying, and bothers us, but it ***also puts your account in danger of being trolled***.
If you put your credentials online, somebody may reset your account, delete your pets/items, access your personal information (i.e. state, grade, IP address, parent's email), change your name/appearance, give negative infinity gold, give glitched pets so that it's impossible to battle, change your password, or even log into your actual email. (List not exhaustive.)

Please do not share your account.

**Comment by ZDLua** [comment]
Nice

**Comment by BoredFishRE** [comment]
Yes. Please stop asking. It's getting very annoying.

**Comment by RtypeX** [comment]
verY tRuE
